src/services/faceRecognition.js

```
@@ -0,0 +1,62 @@
+        detection1.descriptor,
+        detection2.descriptor
+      );
+
+      const similarity = 1 - distance;
+      
+      return {
+        similarity,
+        isMatch: similarity > 0.6
+      };
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)